lib/msal-core/src/PopTokenCommon.ts

```diff
@@ -0,0 +1,61 @@
+export enum HashAlgorithm {
```

Comment:
maybe a src/proof-of-possesion folder that has all your pop pieces, and then you don need a common, you could have a types.ts, utils.ts etc...

Reply:
Agreed.